src/routes/marketplace/+page.svelte

```diff
@@ -0,0 +1,110 @@
+<script lang="ts">
+  import combinedBlends from '$lib/assets/marketplace/combined-blends.png';
+  import { goto } from "$app/navigation"; 
+  import { user, willAttemptLogin } from '$lib/firebase';
+
+  const premadeDecks = [
+    {
+      "name": "Combined Blends", 
+      "description" : "This deck is aimed at blah blah blah a description of what this deck was made for.",
+      "author": "Summer Kiesel",
+      "image": combinedBlends,
+      "id": "-NaT4aMfMtKhI7de8PQb"
+    }
+  ]
+
+  const importMarketplaceDeck = (id: string) => {
+    if (!$user && !willAttemptLogin())
+      goto(`/login?successRedirect=${encodeURIComponent(location.pathname)}`);
+    else if ($user)
+      goto(`/marketplace/${id}`)
+  }
+</script>
+
+<svelte:head>
+  <title>Marketplace</title>
+</svelte:head>
+
+<main>
+  <section class="library-description">
+    <h1>Blend Pro Deck Marketplace</h1>
+    <p>Pro Users have access to a library of premade decks provided by our Blend Team.</p>
+    <p>You can import any of these decks into your own library.</p>
+    <p>Something about how Summer is certified in OG and made these decks to help teachers pinpoint different phonics skills.</p>
+  </section>
+  
+  <section class="decks">
+    {#each premadeDecks as deck}
+    <div class="deck">
+      <div class="side-by-side">
+        <img src={deck.image} alt="sample"/>
+        <section class="description">
+          <h2>{deck.name}</h2>
+          <p>{deck.description}</p>
+          <p><i>Created by: {deck.author}</i></p>
+          <button on:click={ () => importMarketplaceDeck(deck.id) } class="btn">Import</button>
```

Comment:
It's best practice to not have buttons that act like links and vice versa. We can style a link like a button or just put a button nested inside an `<a>` tag, but if clicking it changes the page, it should typically be an `<a>` tag. So here we may want to just calculate the appropriate URL in the script above and then assign it like this:
```suggestion
          <a href={calculatedAddress}>
              <button class="btn">Import</button>
           </a>
```

Reply:
i'm 100% certain you have already told me this and it went in one ear and out the other lmao i will fix it!!